packages/js-client/src/internal/interfaces.ts

```diff
@@ -0,0 +1,35 @@
+import { PrepareInstallationParams, StoreNumberStepValue } from "../types";
+import {
+  DaoAction,
+  GasFeeEstimation,
+  PrepareInstallationStepValue,
+} from "@aragon/sdk-client-common";
+
+export interface ISimpleStorageClient {
+  methods: ISimpleStorageClientMethods;
+  estimation: ISimpleStorageClientEstimation;
+  encoding: ISimpleStorageClientEncoding;
+  decoding: ISimpleStorageClientDecoding;
+}
+
+export interface ISimpleStorageClientMethods {
+  // fill with methods
+  prepareInstallation(
+    params: PrepareInstallationParams,
+  ): AsyncGenerator<PrepareInstallationStepValue>;
+  storeNumber(number: bigint): AsyncGenerator<StoreNumberStepValue>;
+}
+export interface ISimpleStorageClientEstimation {
+  prepareInstallation(
+    params: PrepareInstallationParams,
+  ): Promise<GasFeeEstimation>;
+  storeNumber(
+    number: bigint,
+  ): Promise<GasFeeEstimation>;
```

Comment:
Don't we have a method to retrieve the stored value?

Reply:
Because I forgot

Reply:
Let's add it if possible, please